docs/structured-streaming-programming-guide.md

```diff
@@ -1958,6 +1958,11 @@ Here are the configs regarding to RocksDB instance of the state store provider:
     <td>The waiting time in millisecond for acquiring lock in the load operation for RocksDB instance.</td>
     <td>60000</td>
   </tr>
+  <tr>
+    <td>spark.sql.streaming.stateStore.rocksdb.maxOpenFiles</td>
+    <td>The number of open files that can be used by the RocksDB instance. Value of -1 means that files opened are always kept open.</td>
```

Comment:
What does happen if it hits the limit? Would be great to document that if it fails or do something else.

Reply:
Done - documented the behavior